### DISCUSSION TITLE
brew install mxnet

### DISCUSSION THREAD

**Discussion by ankane** [apache/incubator-mxnet#21142] [General]
Hi, just wanted to share that the C++ package is now available on Homebrew, so Homebrew users can install it with:

```sh
brew install mxnet
```

Homebrew precompiles libraries for Mac x86-64, Mac arm64, and Linux x86-64, so installation is pretty much instant 🎉

Ref: https://github.com/Homebrew/homebrew-core/pull/111872

**Comment by RusFortunat** [comment]
Thank you very much, good Sir!
